scripts/text_reading/beautify_cosmos_with_ftfy.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+import ftfy
```

Comment:
@maxaalexeeva a few things here...

(1) `ftfy` is a package that is available via PyPi. We have an import strategy for our Python code in the repo and that strategy involves separating imports based upon their origin. The schematic for our import strategy is shown below:

```python
import <package-from-python-StdLib>
import <package2-from-python-StdLib >
...
import <package-from-PyPi>
import <package2-from-PyPi>
...
import <package-defined-python-file-in-AutoMATES>
import <package2-defined-python-file-in-AutoMATES>
```
Can you please move the `ftfy` import so that it fits that style?

(2) you should add the import to our `setup.py` file with the other imports for the repository that way new members will have this package installed when they setup their VM. (While you are doing that, I think I forgot to add `fastparquet` to that list. If you could add that package as well that would be great!)